packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/components/Header.astro

```diff
@@ -31,8 +31,9 @@ if ( !name ) {
       label={"Avatar of " + useName}
     ></Avatar> 
     <div>
-      Current User: <br />
-      <b>{useName}</b>
+      Current User: 
+      <Divider />
```

Comment:
Again, are we sure we actually want a divider here? Instead of the "Current User" label just being a block element

Reply:
This is for the Mobile navigation panel (and it is not just a BR element! i swear!)  lol... i am still working to fix the line reset that is going on so i don't have to add the `divider` class on to every single one of them...

There is supposed to be a line between this....
![image](https://github.com/astrolicious/studiocms/assets/30383579/14e0651a-3c70-4627-9188-5fe130c4a0e9)